sur/aquarium.hoon

```diff
@@ -0,0 +1,96 @@
+::
+::  Traditionally, ovo refers an event or card, and ova refers to a list
```

Comment:
ovo refers to an event

Reply:
You're both wrong: `ovo` is the conventional name for an `+ovum`, which is `(pair wire card)`. An "event" is `(pair @da ovum)`.